src/helpers.c

```diff
@@ -636,9 +636,9 @@ int swupd_init(int *lock_fd)
 
 /* this function prints the initial message for all utils
 */
-void copyright_header(const char *name)
+void copyright_header(void)
```

Comment:
This is no longer a "helper", it could move into src/main.c, then made static, removed from the header...

Reply:
true